pkg/transport/generic/cmd/generic-main.go

```diff
@@ -58,6 +60,11 @@ const (
 )
 
 func GenericMain(transportImplementation transport.Transport) {
+
+	go func() {
+		http.ListenAndServe("localhost:6061", nil)
+	}()
```

Comment:
This is already covered by https://github.com/kubestellar/kubestellar/blob/v0.27.1/pkg/transport/generic/cmd/generic-main.go#L75